devices/ePaper/Drivers/Ssd168x/Ssd1681/Ssd1681.cs

```diff
@@ -0,0 +1,295 @@
+
+            // Set Panel Border
+            SendCommand((byte)Command.BorderWaveformControl);
+            SendData(0xc0);
+
+            // Set Temperature sensor to use internal temp sensor
+            SendCommand((byte)Command.TempSensorControlSelection);
+            SendData(0x80);
+
+            // Do a full refresh of the display
+            PerformFullRefresh();
+        }
+
+        /// <inheritdoc/>
+        public override void Flush()
+        {
+            // write B/W and Color (RED) frame to the display's RAM.
+            DirectDrawBuffer(
+                CurrentFrameBufferStartXPosition,
+                CurrentFrameBufferStartYPosition,
+                _frameBuffer2bpp.BlackBuffer.Buffer);
+
+            DirectDrawColorBuffer(
+                CurrentFrameBufferStartXPosition,
+                CurrentFrameBufferStartYPosition,
+                _frameBuffer2bpp.ColorBuffer.Buffer);
+        }
+
+        /// <summary>
+        /// Draws a single pixel to the appropriate frame buffer.
+        /// </summary>
+        /// <param name="x">The X Position.</param>
+        /// <param name="y">The Y Position.</param>
+        /// <param name="inverted">True to invert the pixel from white to black.</param>
+        /// <remarks>
+        /// The SSD1681 comes with 2 RAMs: a Black and White RAM and a Red RAM.
+        /// Writing to the B/W RAM draws B/W pixels on the panel. While writing to the Red RAM draws red pixels on the panel (if the panel supports red).
+        /// However, the SSD1681 doesn't support specifying the color level (no grayscaling), therefore the way the buffer is selected 
+        /// is by performing a simple binary check: 
+        /// if R >= 128 and G == 0 and B == 0 then write a red pixel to the Red Buffer/RAM
+        /// if R == 0 and G == 0 and B == 0 then write a black pixel to B/W Buffer/RAM
+        /// else, assume white pixel and write to B/W Buffer/RAM.
+        /// </remarks>
+        public void DrawPixel(int x, int y, bool inverted)
+        {
+            DrawPixel(x, y, inverted ? Color.Black : Color.White);
+        }
+
+        /// <inheritdoc/>
+        /// <remarks>
+        /// The SSD1681 comes with 2 RAMs: a Black and White RAM and a Red RAM.
+        /// Writing to the B/W RAM draws B/W pixels on the panel. While writing to the Red RAM draws red pixels on the panel (if the panel supports red).
+        /// However, the SSD1681 doesn't support specifying the color level (no grayscaling), therefore the way the buffer is selected 
+        /// is by performing a simple binary check: 
+        /// if R >= 128 and G == 0 and B == 0 then write a red pixel to the Red Buffer/RAM
+        /// if R == 0 and G == 0 and B == 0 then write a black pixel to B/W Buffer/RAM
+        /// else, assume white pixel and write to B/W Buffer/RAM.
+        /// </remarks>
+        public override void DrawPixel(int x, int y, Color color)
```

Comment:
for me, this seems the exact same function as in the 1680. What do I miss?
If it's the same, why not put it in the base class?